hack/make-rules/update.sh

```diff
@@ -78,3 +78,13 @@ done
 ${KUBE_ROOT}/hack/arktos_copyright.sh ${KUBE_ROOT} ${KUBE_ROOT}/_output
 
 echo -e "${color_green:?}Update scripts completed successfully${color_norm}"
+
+echo "Now, let's check whether there is file updated."
+echo "We define return value as 0 if there is no file updated, and 1 if there is file updated."
+MODIFIED_FILES=$(git status | grep modified:) || true
+if [[ "${MODIFIED_FILES:-}" != "" && "${MODIFIED_FILES:-}" != "	modified:   api/openapi-spec/swagger.json" ]]; then
```

Comment:
Why exclude swagger.json? If swagger.json is updated, there is something wrong.

Reply:
Updated